api/reporting/queries/live/latest_table_views/latest_users.sql

```diff
@@ -6,8 +6,8 @@ FROM
 WHERE
         u.snapshot_timestamp = (
         SELECT
-            MAX(u2.snapshot_timestamp)
+            MAX(vs2.snapshot_timestamp)
```

Comment:
I believe these files are now defunct (replaced by Terraform), so we should hopefully be able to remove them eventually. Are they still actually needed for something?

Reply:
Yes, these should be fine to delete. I'm not sure what that docs file is for, but it's probably also safe to delete